lib/java/com/google/android/material/datepicker/res/layout/mtrl_picker_actions.xml

```diff
@@ -24,6 +24,20 @@
   android:gravity="end"
   android:orientation="horizontal">
 
+  <Button
+    android:id="@+id/clear_button"
+    style="?attr/buttonBarNegativeButtonStyle"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center_vertical"
+    android:text="@string/mtrl_picker_clear"/>
+
+  <Space
```

Comment:
Replace containing LinearLayout with ConstraintLayout instead of using weighted space here.